Marlin/language.h

```diff
@@ -129,6 +129,9 @@
 #define MSG_COUNT_A                         " Count A: "
 #define MSG_ERR_KILLED                      "Printer halted. kill() called!"
 #define MSG_ERR_STOPPED                     "Printer stopped due to errors. Fix the error and use M999 to restart. (Temperature is reset. Set it after restarting)"
+#define MSG_BUSY_SIGNAL                     "Busy"
+#define MSG_WAITING_FOR_USER                "Paused for user"
+#define MSG_WAITING_FOR_INPUT               "Paused for input"
 #define MSG_RESEND                          "Resend: "
```

Comment:
Maybe a "Busy: " prefix could simplify parsing for the host - it had to check for only one expression.
Anything containing a "...wait..." is not a good idea - already used.